src/lib/browse/layers/points/CriticalIssues.svelte

```diff
@@ -138,16 +75,78 @@
   {/if}
 </CollapsibleCard>
 
-<InteractiveLayer
-  layer={source + "-points"}
-  {tooltip}
-  {show}
-  clickable={false}
-/>
-<InteractiveLayer
-  layer={source + "-clusters"}
-  {show}
-  clickable
-  on:click={clickCluster}
-  ignoreHigherLayers={[source + "-counts"]}
-/>
+<GeoJSON
+  id={source}
+  data={gj}
+  cluster={{
+    radius: 500,
```

Comment:
I can't figure out why, but the clustering is very different than before. I bumped up the radius to try and compensate, but it's still odd. I'll file a bug to follow up and improve it